crypto/keyring_test.go

```diff
@@ -231,3 +233,40 @@ func TestKeyringCapabilities(t *testing.T) {
 	assert.True(t, keyRingTestMultiple.CanVerify())
 	assert.True(t, keyRingTestMultiple.CanEncrypt())
 }
+
+func TestVerificationTime(t *testing.T) {
+	message := NewPlainMessageFromString("Hello")
+	pgp.latestServerTime = 1632312383
```

Comment:
Would you mind resetting the time after the test is done (with a defer) back to `testTime`?

Reply:
Do I need a defer, if I do `pgp.latestServerTime = testTime` right after ?
Only thing I see that would avoid that is panics.

Reply:
Yeah, and a panic interrupts all tests. It's fine without any defer, as long as there is no fail in the middle.

Reply:
I added a defer, it's more "the golang way"